src/XMakeTasks/Strings.resx

```diff
@@ -1562,12 +1562,6 @@
         LOCALIZATION: Please don't localize "CopyLocal" this is an item meta-data name.
    </comment>
   </data>
-  <data name="ResolveAssemblyReference.NoBecauseParentReferencesFoundInGac">
-    <value>This reference is not "CopyLocal" because the CopyLocalDependenciesWhenParentReferenceInGac property is set to false and all the parent references for this reference are found in the GAC.</value>
-    <comment>
-        LOCALIZATION: Please don't localize "CopyLocal" this is an item meta-data name. "CopyLocalDependenciesWhenParentReferenceInGac" is a property name.
-   </comment>
-  </data>
```

Comment:
I tried, but bailed after ~15min. Some of the edits are in files that I also moved, making everything a pain. I vote for not splitting that monster commit :)